pom.xml

```diff
@@ -24,18 +24,21 @@
         <module>ui.content</module>
     </modules>
 
-    <url>https://github.com/cid15/aem-groovy-console</url>
-
-    <organization>
-        <name>CID 15</name>
-        <url>https://www.cid15.org</url>
-    </organization>
+    <url>https://github.com/orbinson/aem-groovy-console</url>
 
     <developers>
         <developer>
             <name>Mark Daugherty</name>
             <email>mark.r.daugherty@gmail.com</email>
         </developer>
+        <developer>
+            <name>Barry d'Hoine</name>
+            <email>barry.dhoine@gmail.com</email>
```

Comment:
```suggestion
            <email>barrydhoine@gmail.com</email>
```

Reply:
should work just the same :D

Reply:
I know but I always use without dot for print xD b.a.r.r.y.dhoine+roywhyyoudothistome@gmail.com also works but that does not validate to use it 😀